### ISSUE TITLE
Closes when looking for "0" as Int32 values on big applications

### ISSUE THREAD

**Issue by Pensantos** [scanmem/scanmem#331]
Hangs for a few seconds then just closes. Application does not crash, just Game Conqueror. 
Tested on Ubuntu 19.04 and 18.04, Lubuntu 19.04 and 18,10, Ubuntu mate 19.04.

**Comment by 12345ieee** [comment]
Uhm, can you open it in a terminal and paste here the output?

**Comment by Pensantos** [comment]
```
info: maps file located at /proc/25545/maps opened.
info: 73 suitable regions found.
info: maps file located at /proc/25545/maps opened.
info: 73 suitable regions found.
info: maps file located at /proc/26082/maps opened.
info: 208 suitable regions found.
info: maps file located at /proc/26082/maps opened.
info: 208 suitable regions found.
info: maps file located at /proc/26082/maps opened.
info: 439 suitable regions found.
......................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................error: 
Killed by signal 11.
```

**Comment by 12345ieee** [comment]
I'm pretty sure this is an "out of memory" error.
scanmem uses roughly 4 bytes per match, most of the memory of any application is zeroes, you've probably finished your memory.

This is then a duplicate of #307 .

**Comment by MichaelElianKevin** [comment]
How to fix "out of memory"? I didn't search " 0", i searched "127" and than got "Killed by signal 11"

**Comment by 12345ieee** [comment]
Closed as duplicate of #307 .
